pkgs/development/python-modules/ibis-framework/default.nix

```diff
@@ -181,6 +181,8 @@ buildPythonPackage rec {
   };
 
   meta = with lib; {
+    # Not compatible with pyarrow13 yet.
+    broken = true;
```

Comment:
Is it possible to simply skip or ignore the failing tests? Most of the ibis backends will work with pyarrow 13.

Reply:
Done.